### ISSUE TITLE
Background Image with no alt text

### ISSUE THREAD

**Issue by colbrydi** [colbrydi/jupyterinstruct#3]
When I test this on the MSU jupyterhub server I get an extra image with no alt text.  Searching the html shows the image is a background image of some sort.  I either need to remove the image, find a way to add an alt text or ignore the error inside of the validate function.  TO help with search the beginning of the image start with the following:

```data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABCoAAAK6CAYAA```

**Comment by colbrydi** [comment]
I am able to reproduce this bug on jupyterhub-grader.egr.msu.edu.  This is one of the newer systems so I wonder if this is a newer bug.  We should create a environment and run some tests.
